src/components/ProductListContainer/index.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import ProductCard from '../ProductCard'
+import './styles.css'
+import PropTypes from 'prop-types'
+
+const fakeProductList = [
+  {
+    id: 1,
+    productPhotoURL: 'https://placehold.co/308x210',
+    listPrice: 100,
+    discountedPrice: 80,
+    description: 'Este es un producto',
+    colors: 'rojo, verde, negro',
+    occupancyPercentage: 50,
+    groupCapacity: 10,
+    suscriptorsNumber: 5,
+    expirationDate: new Date().toLocaleTimeString(),
+    discountPercentage: 20
```

Comment:
This is fake data but it should match with the `Model` (the real data representation).

For example, the `id` here and in the PropTypes is a Number. That could happen, but the majority of cases ID's are UUID's, for example this one `4f452c6c-4d2f-47d5-83b4-45ac87ab988e`. This combinations of letters and numbers can be represented with a String. I advise that you change the PropTypes of it to a String, but for the fake data you can use the strings `'1'`, `'2'`.

For `expirationDate`, we should use the standard iso string `new Date().toISOString()` instead.

I attach a comparison

![image](https://user-images.githubusercontent.com/35865469/104226899-4a2f3e80-5427-11eb-9917-bd18d4e3c648.png)